sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/directconnectivity/ReplicatedResourceClientGoneForWriteTest.java

```diff
@@ -0,0 +1,127 @@
+package com.azure.cosmos.implementation.directconnectivity;
```

Comment:
Not sure why the CI didn't catch this, but this is missing Microsoft copyright header information.

Reply:
Fixed in next iteration